deepchecks/checks/integrity/new_label.py

```diff
@@ -100,11 +101,13 @@ def add_condition_new_label_ratio_not_greater_than(self, max_ratio: float = 0):
         def new_category_count_condition(result: Dict) -> ConditionResult:
             if result:
                 column_name = result['column_name']
+                new_labels = result['new_labels']
                 new_label_ratio = result['n_new_labels_samples']/result['n_samples']
                 if new_label_ratio > max_ratio:
+
                     return ConditionResult(False,
-                                           f'Found more than {format_percent(max_ratio)} new labeled data in'
-                                           f' label column: {column_name}')
+                                           f'Found {format_percent(new_label_ratio)} new labeled data in'
```

Comment:
Can improve the wording. maybe
"Found new labels {new_labels} in test data, making {format_percent(new_label_ratio)} of test data."
Not sure what info is there in {column_name}